app/components/form/SideModalForm.tsx

```diff
@@ -30,14 +30,6 @@ type EditFormProps = {
 
 type SideModalFormProps<TFieldValues extends FieldValues> = {
   form: UseFormReturn<TFieldValues>
-  /**
-   * A function that returns the fields.
-   *
-   * Implemented as a function so we can pass `control` to the fields in the
-   * calling code. We could do that internally with `cloneElement` instead, but
-   * then in the calling code, the field would not infer `TFieldValues` and
-   * constrain the `name` prop to paths in the values object.
-   */
```

Comment:
This comment has been wrong for two years 😁 

https://github.com/oxidecomputer/console/blame/eed4a4e7873e1d7511cfe66c7ed9f16e916f2ae1/app/components/form/SideModalForm.tsx#L33-L41